html/header.php

```diff
@@ -69,20 +70,46 @@
 				width: 100% !important;
 			}
 		</style>
+		<script src="./js/color-toggle.js" defer>
 	</head>
 	<body onUnload="document.getElementById('secret').value = ''">
-		<script>
+	<script>
 			if (location.protocol != 'https:') {
 				document.write('<div style="padding-top: 1%" class="container"><div class="alert alert-danger"><strong>Danger!</strong> This site is not being accessed over an encrypted connection. Do NOT input any sensitive information!</div></div>');
 			}
-			function copyText() {
-				var textToCopy = document.getElementById("copy");
-				textToCopy.select();
-				document.execCommand("copy");
-			}
 		</script>
-		<?php
+		<header>
+			<nav class="navbar navbar-expand-lg">
+		  		<div class="container-fluid">
+		    		<a class="navbar-brand" href="#">
+						<?php
+							if ( $settings['site_logo'] != '' ) {
+								echo '<img src="'. $settings['site_logo'] .'" alt="Logo" width="200" class="d-inline-block align-middle">';
+							}
+						?>
+						<?php
+							if ( $settings['display_title'] == 'true') {
+								echo '' . $settings['site_title'] . '';
+							} else {
+								echo '';
+							}
```

Comment:
Why are we echoing out empty strings here? Can we do something like this instead?

```
if ( $settings['display_title'] == 'true') {
    echo $settings['site_title'];
}
```

Reply:
fixed in [3ce241d](https://github.com/AndrewPaglusch/FlashPaper/pull/159/commits/3ce241dd1de608ab90fad606d6ad1d0ef2adaf5a)